modules/xmpp/Caps.js

```diff
@@ -102,10 +106,17 @@ export default class Caps extends Listenable {
      * @param {String} feature the name of the feature.
      * @param {boolean} submit if true - new presence with updated "c" node
      * will be sent.
+     * @param {boolean} external whether this feature was added externally to the library.
```

Comment:
Can you elaborate on what makes a feature "external"?

Reply:
External to the library ... Like remote control, jibri ... These are two we set from jitsi-meet. I will try updating the js doc.

Reply:
👍